code/datums/martial/krav_maga.dm

```diff
@@ -1,189 +1,148 @@
-	to_chat(A, span_danger("You pound [D] on the chest!"))
-	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
-	if(D.losebreath <= 10)
-		D.losebreath = clamp(D.losebreath + 5, 0, 10)
-	D.adjustOxyLoss(10)
-	log_combat(A, D, "quickchoked", name)
+/datum/martial_art/krav_maga/proc/neck_chop(mob/living/user, mob/living/target)
+	if(!iscarbon(target))
+		return FALSE
+	var/mob/living/carbon/carbon_defender = target
+	target.visible_message(span_warning("[user] karate chops [target]'s neck!"), \
+					span_userdanger("Your neck is karate chopped by [user], rendering you unable to speak!"), span_hear("You hear a sickening sound of flesh hitting flesh!"), COMBAT_MESSAGE_RANGE, user)
+	to_chat(user, span_danger("You karate chop [target]'s neck, rendering [target.p_them()] unable to speak!"))
+	playsound(get_turf(user), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+	target.apply_damage(5, user.get_attack_type())
+	log_combat(user, target, "neck chopped", name)
+	carbon_defender.silent = 10 //10 life() ticks without the ability to speak
 	return TRUE
 
-/datum/martial_art/krav_maga/proc/neck_chop(mob/living/A, mob/living/D)
-	D.visible_message(span_warning("[A] karate chops [D]'s neck!"), \
-					span_userdanger("Your neck is karate chopped by [A], rendering you unable to speak!"), span_hear("You hear a sickening sound of flesh hitting flesh!"), COMBAT_MESSAGE_RANGE, A)
-	to_chat(A, span_danger("You karate chop [D]'s neck, rendering [D.p_them()] unable to speak!"))
-	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
-	D.apply_damage(5, A.get_attack_type())
-	if (iscarbon(D))
-		var/mob/living/carbon/carbon_defender = D
-		if(carbon_defender.silent <= 10)
-			carbon_defender.silent = clamp(carbon_defender.silent + 10, 0, 10)
-	log_combat(A, D, "neck chopped", name)
+
+/datum/martial_art/krav_maga/proc/martial_strikes(mob/living/user, mob/living/target)
+	if(!iscarbon(target))
+		return FALSE
+
+	//I don't really know what I'm describing myself tbh, but it does less damage and more stamina so I went with this
+	target.visible_message(span_warning("[user] deftly strikes at [target] with an open palm!"), \
+					span_userdanger("[user] attacks you with an open palm!"), span_hear("You hear a sickening sound of flesh hitting flesh!"), COMBAT_MESSAGE_RANGE, user)
+	to_chat(user, span_danger("You strike at [target] with an open palm!"))
+	playsound(get_turf(user), 'sound/effects/hit_punch.ogg', 50, 1, -1)
+
+	var/obj/item/bodypart/affecting = target.get_bodypart(ran_zone(user.get_combat_bodyzone(target)))
+	var/limb_armor = target.run_armor_check(affecting, MELEE)
+
+	//Minor damage, high stamina for an unarmed strike but still not great. This is intended as a "better than nothing" if officer is disarmed.
+	target.apply_damage(2, user.get_attack_type(), affecting, limb_armor)
+	target.apply_damage(17, STAMINA, affecting, limb_armor)
 	return TRUE
 
-/datum/martial_art/krav_maga/grab_act(mob/living/A, mob/living/D)
```

Comment:
Not change you need to make, but I made a mistake with this grab_act. 

It made it possible to perform krav maga moves on people with items in your hand, when prior to combat mode you needed to do it with an empty hand.

Add that as a fix to the CL

Reply:
Done!